bridges/GoogleScholarBridge.php

```diff
@@ -1,20 +1,102 @@
 <?php
 
-class GoogleScholarBridge extends BridgeAbstract
+class GoogleScholarV2Bridge extends BridgeAbstract
 {
-    const NAME = 'Goolge Scholar';
+    const NAME = 'Google Scholar v2';
     const URI = 'https://scholar.google.com/';
-    const DESCRIPTION = 'Follow authors of scientific publications.';
-    const MAINTAINER = 'thefranke';
+    const DESCRIPTION = 'Search for publications or follow authors on Google Scholar.';
+    const MAINTAINER = 'nicholasmccarthy@github';
```

Comment:
Maintainer name is just the Github name without the "@github" appended

Reply:
Sorted. Also, hello! Didn't mean to jump over your maintainer status but this was a holdover from when it was a separate file, and didn't see examples of multiple maintainers/maintainer history.

Reply:
No worries, I want a cool Bridge, don't really care abou the maintainer name. We'll do this together 🫡